### PR TITLE
Fix datasource documentation

### PR DESCRIPTION
**Pull request by ingobecker**
Fixed the datasource issue but kept the old pre 0.12 syntax.

### CODE REVIEW

File: website/docs/d/storage.html.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Get the id of a storage resource. This can be used to link storages to a server.
 Using the storage datasource for the creation of a server:
 
 ```hcl
-resource "gridscale_storage" "storagename"{
+data "gridscale_storage" "storagename"{
 	name = "terraform-storage"
 	capacity = 10
```
Comment (itakouna) [inline website/docs/d/storage.html.md]: `capacity` should not be part of datasource.

**Comment by itakouna** [comment]
you will get `Error: Unsupported argument` `An argument named "name" is not expected here.` I've looked into datasource implementation, and I found it's not really functional :).
e,g, https://github.com/gridscale/terraform-provider-gridscale/blob/27f919b59d6183185ecc4597475a8f042a5d9fad/gridscale/datasource_gridscale_network.go#L10,  it did not get any argument such as `name`.  I think we have to fix this.

File: website/docs/d/network.html.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ resource "gridscale_server" "servername"{
 
 The following attributes are exported:
 
-* `id` - The UUID of the network.
\ No newline at end of file
+* `id` - The UUID of the network.
```
Comment (itakouna) [inline website/docs/d/network.html.md]: It should return other properties e.g, name.

**Comment by ingobecker** [comment]
What's about the other datasources? Do they have the same `Error: Unsupported argument` problem?

**Comment by itakouna** [comment]
I did not test all. However, we can check if datasources accept arguments. I think the only working one is the template https://github.com/gridscale/terraform-provider-gridscale/blob/27f919b59d6183185ecc4597475a8f042a5d9fad/gridscale/datasource_gridscale_template.go#L16.

**Comment by MarcHarriss** [comment]
Closed to archive the repo, @itakouna, @ingobecker please make sure this is opened again in the official upstream. https://github.com/terraform-providers/terraform-provider-gridscale/

**Pull request closed without merging**